.github/workflows/build_release.yml

```diff
@@ -0,0 +1,72 @@
+name: Build Release
+description: Build release bundle and upload to GitHub release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  build-release:
+    name: Build Release
+    runs-on: macos-26
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Select Xcode version
+        run: sudo xcodes select 26.1.1
+
+      - name: Build Release Archive
+        env:
+          NSUnbufferedIO: YES
+        run: |
+          cmd="\
+            xcodebuild archive \
+              -project Cilicon.xcodeproj \
+              -scheme Cilicon \
+              -configuration Release \
+              -archivePath Cilicon.xcarchive \
+              -destination 'generic/platform=macOS' \
+              CODE_SIGN_IDENTITY='' \
+              CODE_SIGN_STYLE='Manual' \
+              CODE_SIGNING_REQUIRED=NO \
+              CODE_SIGNING_ALLOWED=NO \
```

Comment:
Lets create an unsigned app bundle for now. We can implement notarization later, if required.